packages/gatsby-transformer-cloudinary/gatsby-node.js

```diff
@@ -244,6 +248,40 @@ exports.onCreateNode = async ({
     });
   }
 };
+// The Old Way
+// onPreInit is setting 🌐 global state
+// so that other functions can access that global state. Which is The Old Way. And used here 😉
+
+let coreSupportsOnPluginInit = 'unstable' | 'stable' | undefined;
```

Comment:
The `'unstable' | 'stable' | undefined;` seems to be ts code, and not js code.
Change it to `let coreSupportsOnPluginInit = undefined;`.

Reply:
Is this `let coreSupportsOnPluginInit = 'unstable' | 'stable' | undefined; ` just to explain in the docs? And not part of our actual code?